adal/build.gradle

```diff
@@ -0,0 +1,167 @@
+apply plugin: 'com.android.library'
+// This plugin publishes adal in to the local maven repo
+apply plugin: 'com.github.dcendents.android-maven'
+//These plugins are used for static code analysis tools
+apply plugin: 'findbugs'
+apply plugin: 'pmd'
+apply plugin: 'checkstyle'
+
+group = 'com.microsoft.aad'
+version = '1.1.16'
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+            classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'
+    }
+}
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
+
+    defaultConfig {
+        minSdkVersion 18
+        targetSdkVersion 21
+        versionCode 1
+        versionName version
+        project.archivesBaseName = "adal"
+        project.version = android.defaultConfig.versionName
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+    sourceSets {
+        main {
+            manifest.srcFile 'src/main/AndroidManifest.xml'
+        }
+    }
+
+    lintOptions {
+        abortOnError false
```

Comment:
Hi @agamemnum , thank you for your review. The deployment of the gradle build is still under construction. The flag is set to be false temporarily, and I am fixing the lint errors these days. All the flags of code analysis tools will be set back to be true after finishing fixing all errors reported (ASAP). @RandalliLama